src/pages/basics/introduction.js

```diff
@@ -92,13 +90,12 @@ const IntroductionPage = () => (
       >
         Unified Shader Model
       </a>
-      , where every shader type is practically the same in the functions and
-      APIs they have access to. This makes them simpler to implement in hardware
-      and easy to program in software as well.
+      , where every shader type is practically the same. 
+      This makes them simpler to implement in hardware and easy to program in software as well.
     </p>
     <h3>What do I need to know before getting started?</h3>
     <p>
-      A beginner understanding of matrix and vector math will be helpful when
+      (Make this a list)A beginner understanding of matrix and vector math will be helpful when
```

Comment:
How about

Go through {link} of {link} to learn about the necessary details vector and matrix math, even if you have an understanding of it. This information is vital as it provides a base for understanding all the math that'll be used later on.

As a side note...

Reply:
Since there isn't much of a pre-requisite, a list shouldn't be necessary

Reply:
That would do, yes.